### ISSUE TITLE
dataset label question

### ISSUE THREAD

**Issue by wonchul-kim** [ESandML/FCBFormer#6]
Within the dataset you are mentioning, the mask pixel values are more than 1, even though the number of classes is one.

I though that there would be 0 and 1 values for the pixel of mask image. 

Is it OK?

Could you explain??

**Comment by ESandML** [comment]
Hi, is the mask pixel value for the positive case, to which you are referring, 255? If so, this is just for visualisation of the mask in uint8 format, so that the negative class appears black and the positive class appears white. If the value was 1, it would be almost black and difficult to see. We map these values to 1 using the `ToTensor()` transform, which scales the values from [0, 255] to [0, 1] if the data is in uint8 format.

**Comment by wonchul-kim** [comment]
I meant, for the mask image, the pixel value should be one or zero, but there are other values too. Since the dataset has binary classes (background and polyp), I think there should be two pixel value 1 or 0 in mask image. 

However, when I print the values of mask image from the dataloader in the code, there are many other values. 

Could you explain?

**Comment by ESandML** [comment]
Apologies, I had forgotten about this issue. Some pixels in the masks in the raw format do indeed have values other than 0 and 255, and after being mapped to [0, 1] through the `ToTensor()` transform are also not 0 or 1. Unfortunately, I do not know exactly why this is, you would need to ask the authors of the dataset. We concluded that since the values are always very close to 0 or 1, that this would cause little issue during training. However, in both the evaluation performed during validation (using our implementation of the Dice score given in `Metrics/performance_metrics.py`) and in the final evaluation of the model (`eval.py`), we binarized both the prediction and the ground truth using `x > 0.5`.

**Comment by xhlho** [comment]
I am new to annotation. For tags, what is the appropriate format for me to use labelme, such as TXT or JSON? Could you please take some time to answer, thank you.
